retroshare-gui/src/gui/gxsforums/CreateGxsForumMsg.cpp

```diff
@@ -337,6 +340,22 @@ void  CreateGxsForumMsg::loadFormInformation()
 	//ui.forumMessage->setText("");
 }
 
+static const uint32_t MAX_ALLOWED_GXS_MESSAGE_SIZE = 199000;
```

Comment:
Already defined constant.
Prefer adding method to get it like was done for chat:
ChatWidget::maxMessageSize()
p3Msgs::getMaxMessageSecuritySize(int type)
p3ChatService::getMaxMessageSecuritySize(int type)